docs/src/guides/drupal9/redis.md

```diff
@@ -1,149 +1,154 @@
 ---
 title: "Using Redis with Drupal 9.x"
 sidebarTitle: "Redis"
-description: |
-    Add Redis caching to your existing Drupal site.
+description: Add Redis caching to your existing Drupal site.
 weight: -70
 ---
 
-If you are using the Platform.sh-provided Drupal template, most of this work is already done for you.
-Redis is already configured and is enabled after the installation is complete.
+Redis is a fast open-source in-memory database and cache, 
+useful for application-level caching. 
+For more information on this service, see the [dedicated Redis page](../../add-services/redis.md) 
+or the [official Redis documentation](https://redis.io/docs/).
 
-Note that this `redis` service is ephemeral, meaning it doesn't persist if the container moves or is shut down.
-Your app must treat it as ephemeral and not rely on it being there.
+Follow the instructions on this page to do one of the following:
 
-There are two possible solutions:
+- Add and configure Redis for Drupal 9.x if you have deployed Drupal manually.
+- Fine-tune your existing configuration if you have deployed Drupal 9 using a [Platform.sh template](../../development/templates.md).
 
-* Change the service to persistent Redis (`redis-persistent`).
-* Clean the cache each time your app starts in the `start` key in [your web configuration](../../create-apps/app-reference.md#web-commands).
+## Before you begin
 
-If you are working from an older repository or migrating a pre-built site to Platform.sh, see the instructions below.
+You need:
 
-## Requirements
+- A [Drupal 9 version deployed on Platform.sh](../drupal9/deploy/_index.md)
+- The [Platform.sh CLI](../../administration/cli/)
+- [Composer](https://getcomposer.org/)
+- The [Platform.sh Config Reader library](../../guides/drupal9/deploy/customize.md#install-the-config-reader)
 
-### Add a Redis service
+You also need a `settings.platformsh.php` file from which you can [manage the configuration of the Redis service](../drupal9/deploy/customize.md#settingsphp). 
+If you installed Drupal 9 with a template, this file is already present in your project.
```

Comment:
I also figured this could be moved because it's a slightly different kind of prerequisite given that the file is provided for those who use the template + it helps make the list more "digestible".

Reply:
Good idea